nuxt_src/pages/extra-staff/index.vue

```diff
@@ -3,88 +3,93 @@
-          <a href="https://docs.google.com/forms/d/e/1FAIpQLSeoh2Lgd3LiMxKRjdLtu9Ul1DGZLN54srPdbvIjWoAsFz-rAA/viewform" target="_blank" rel="noopener" class="section_btn section_btn-l">当日スタッフに応募する</a>
+          <a href="https://forms.gle/SkbscteHs4nzGPZB8" target="_blank" rel="noopener" class="section_btn section_btn-l">スタッフに応募する</a>
         </div>
       </div>
     </div>
     <p class="catch">
-      運営にご興味のある方、一緒にScalaMatsuriを盛り上げていきたい方は是非当日スタッフとしてのご参加も検討下さい。
+      カンファレンス運営に興味のある方、Scalaコミュニティを盛り上げていきたい方は、是非当日スタッフとしてのご参加を検討下さい。
     </p>
 
     <div class="section">
       <h2 class="section_title">
-        <span class="section_title_inner">当日スタッフ概要</span>
+        <span class="section_title_inner">直前期スタッフ概要</span>
       </h2>
       <div class="section_table1">
         <dl>
-          <dt>開催日（予定）</dt>
-          <dd><em>2020年5/14(木) 〜 5/16(土)</em><br>前日準備を含めて最大三日間の予定です。また、当日の作業の事前説明会も開催予定です。詳細は決まり次第、追ってご連絡します。</dd>
+          <dt>開催日</dt>
+          <dd><em>2020年10/17(土) 〜 10/18(日)</em><br>当日の作業やZoom Webinarの扱い方について事前説明会も開催します。</dd>
         </dl>
         <dl>
-          <dt>開催時間（予定）</dt>
-          <dd><em>00:00 〜 99:00</em></dd>
+          <dt>開催時間</dt>
+          <dd><em>9:30 〜 19:00</em>(懇親会参加の場合は21:00)</dd>
         </dl>
         <dl>
-          <dt>開催地（予定）</dt>
-          <dd><em>国際交流館</em>（ <a href="https://goo.gl/maps/6CVQ4ZvJMDS2" target="_blank" rel="noopener">Google Maps</a> ）</dd>
+          <dt>開催地</dt>
+          <dd><em>ScalaMatsuri DiscordサーバーおよびZoom Webinar</em></dd>
         </dl>
         <dl>
           <dt>応募方法</dt>
-          <dd><em>応募フォームよりご連絡のつくメールアドレスを入力して下さい。</em><br>後日、こちらでご入力いただいたメールアドレス宛に詳細のご連絡や、Slackのご招待を送ります。招待をお送りするのは開催一ヶ月前ぐらいになる予定です。</dd>
+          <dd>
+            <em>応募フォームよりご連絡のつくメールアドレスを入力して下さい。</em><br>
+            後日、こちらでご入力いただいたメールアドレス宛に詳細のご連絡や、Slackのご招待を送ります。
+          </dd>
         </dl>
+
         <dl>
-          <dt>お手伝い内容（シフト制）</dt>
+          <dt>スタッフ業の内容</dt>
           <dd>
+            <p>事前（希望者のみ）</p>
+            <ul>
+              <li>スライドの和訳字幕付け</li>
+              <li>Webサイト改修（Nuxt.js @ Firebase Hosting)</li>
+            </ul>
+            <p>カンファレンス当日</p>
+            <p>(シフト時間外は自由にScalaMatsuriを楽しんでください！)</p>
             <ul>
-              <li>前日準備、後片付け</li>
-              <li>懇親会準備</li>
-              <li>受付</li>
-              <li>お菓子やコーヒーの補充</li>
-              <li>セッションタイムキーパー</li>
-              <li>お弁当の配布</li>
+              <li>各セッションのタイムキーパー、司会</li>
+              <li>アンカンファレンス</li>
             </ul>
-            <p>シフト時間外は自由にScalaMatsuriを楽しんでください！</p>
           </dd>
         </dl>
         <dl>
-          <dt>当日スタッフ特典(予定)</dt>
+          <dt>スタッフ特典</dt>
           <dd>
             <ul>
-              <li>チケット代無料+チケットに付く特典類(Tシャツは希望サイズをお渡し出来ない場合があります)</li>
-              <li>スタッフTシャツ(ただし、サイズがない場合はお渡し出来ない場合があります)</li>
+              <li>チケット代無料</li>
+              <li><a href="https://scalamatsuri.official.ec/">Tシャツ</a>割引クーポン(システム下限の50円となります)</li>
```

Comment:
suggestionであればPR作成者以外でもコード修正提案を、PR作成者の負担を最低限にした上でできるので、そちらで指摘しても良いかもしれないです

```suggestion
              <li><a href="https://scalamatsuri.official.ec/" target="_blank" rel="noopener">Tシャツ</a>割引クーポン(システム下限の50円となります)</li>
```

こんな感じに(この修正が必要なければ無視して頂いて大丈夫です。)